### PR TITLE
Modify the description of TLS related parameters in the lightning doc…

### PR DESCRIPTION
**Pull request by ljun0712**
### What is changed, added or deleted? (Required)

The document description is incorrect and needs to be adjusted.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Comment by D3Hunter** [comment]
/hold

current doc is correct

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ summary: 使用命令行配置 TiDB Lightning。
 | --analyze *level* | 导入后分析表信息，可选值为 required、optional（默认值）、off | `post-restore.analyze` |
 | --checksum *level* | 导入后比较校验和，可选值为 required（默认值）、optional、off | `post-restore.checksum` |
 | --check-requirements *bool* | 任务开始之前检查集群版本兼容性，以及运行过程中检查 TiKV 的可用存储空间是否大于 10%（默认值为 true）| `lightning.check-requirements` |
-| --ca *file* | TLS 连接的 CA 证书路径 | `security.ca-path` |
-| --cert *file* | TLS 连接的证书路径 | `security.cert-path` |
-| --key *file* | TLS 连接的私钥路径 | `security.key-path` |
+| --ca *file* | 与 TiDB Server 建立 TLS 连接的 CA 证书路径 | `tidb.security.ca-path` |
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-command-line-full.md:36]: 是 TiDB Lightning 与 TiDB Server 建立 TLS 连接的 CA 证书路径吗

File: tidb-lightning/tidb-lightning-command-line-full.md
Comment (qiancai) [inline tidb-lightning/tidb-lightning-command-line-full.md:1]: 这个修改是否适用于  v8.2 和之后版本呢？看到 PR 提到了 release-8.1 分支上

File: tidb-lightning/tidb-lightning-command-line-full.md (hunk 1 above)
Comment (ljun0712) [reply tidb-lightning/tidb-lightning-command-line-full.md:36]: 是的

Comment (D3Hunter) [reply tidb-lightning/tidb-lightning-command-line-full.md:36]: https://github.com/pingcap/docs-cn/pull/17844#issuecomment-2203118920

**Pull request closed without merging**